.dockerignore

```diff
@@ -115,8 +115,6 @@ dmypy.json
 .pyre/
 
 # Config and data
-config/*.yaml
```

Comment:
I suggest to add `config/settings.yaml` because it contains some sensitive data

Reply:
This is just the .dockerignore. This way, when creating a docker image locally, the _config/settings.yaml_ will be copied on the docker image, allowing the user to easily transfer settings